### ISSUE TITLE
did anyone use IM-Tornado with V3 (2x300) myseq data?

### ISSUE THREAD

**Issue by suzumar** [pjeraldo/imtornado2#9]
in which case do you have the parameters for max lengths?

**Comment by cjfields** [comment]
Yes, we have had to adjust parameters to deal with variation in read length and quality scores from samples.  We typically run FASTQC per sample but generate a single report using MultiQC to assess overall quality, then try to pick reasonable cutoffs.

EDIT: I should add, this was on V4 data, not V3.
